pyproject.toml

```diff
@@ -11,7 +11,7 @@ package-mode = false
 python = "^3.12"
 discord-py = "^2.4.0"
 python-dotenv = "^1.0.1"
-redis = "^5.2.0"
+redis = "^5.2.1"
```

Comment:
Redis is no longer being used as we're not using upstash as the database solution anymore. This is a good reason why we should review all the dependencies being used regularly